Lombiq.VueJs.Samples/wwwroot/vue/demo-repeater.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Why isn't `wwwroot` ignored? Is it related to NuGet packaging?

Reply:
Yes.